raiden/network/proxies/token_network.py

```diff
@@ -468,7 +470,7 @@ def can_transfer(
             self,
             participant1: typing.Address,
             participant2: typing.Address,
-            channel_identifier: typing.ChannelID = None,
+            channel_identifier: typing.ChannelID,
```

Comment:
removing `None` as default argument should be safe as the only place this is used is [here](https://github.com/raiden-network/raiden/blob/023554420534eb2ea8a1705bb7653e9f44d7d214/raiden/network/proxies/payment_channel.py#L158) and it's already provided.